src/core/utils/base64-utils.ts

```
@@ -0,0 +1,26 @@
+    // Convert from Base64URL to standard Base64
+    let base64 = base64url.replace(/-/g, "+").replace(/_/g, "/");
+    // Pad with '=' to make length a multiple of 4
+    const pad = base64.length % 4;
+    if (pad !== 0) {
+      base64 += "=".repeat(4 - pad);
+    }
+    const binary = atob(base64);
+
+    return binary;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)